blog.html

```diff
@@ -0,0 +1,84 @@
+<!DOCTYPE html>
+<html lang="en">
+	<head>
+    	<meta charset="utf-8">
+    	<meta name="description" content="relaxr blog">
+    	<meta name="blog" content="one, two, three">
+
+		<title>Relaxr Blog</title>
```

Comment:
if i change "tab size" to 2 on the bottom right of the window in sublime does this correct this?